android/java/org/chromium/chrome/browser/BraveRewardsPanelPopup.java

```diff
@@ -1825,8 +1819,11 @@ public void onClick(View v) {
                 case BraveRewardsExternalWallet.CONNECTED:
                 case BraveRewardsExternalWallet.PENDING:
                 case BraveRewardsExternalWallet.VERIFIED:
-                    if (walletBalance < WALLET_BALANCE_LIMIT && !isVerifyWalletEnabled()) {
-                        Toast.makeText(ContextUtils.getApplicationContext(), root.getResources().getString(R.string.required_minium_balance), Toast.LENGTH_SHORT).show();
+                    if (walletBalance < WALLET_BALANCE_LIMIT
+                            && mBraveRewardsNativeWorker.getExternalWalletType().equals(
```

Comment:
Instead of getting the wallet type from `RewardsService`, we should probably get the wallet type from the `BraveRewardsExternalWallet` object (it should probably have a "type" member on it).

On the desktop side, the `mojom::ExternalWallet` struct has a `type` field that we use for this kind of thing.

Reply:
Thanks @zenparsing updating it now.